wpilibc/src/main/native/cpp/simulation/SingleJointedArmSim.cpp

```diff
@@ -19,24 +19,28 @@ SingleJointedArmSim::SingleJointedArmSim(
     const LinearSystem<2, 1, 1>& system, const DCMotor& gearbox, double gearing,
     units::meter_t armLength, units::radian_t minAngle,
     units::radian_t maxAngle, bool simulateGravity,
+    units::radian_t startingAngleRad,
```

Comment:
Don't add a unit postfix for C++ variables. The type already encodes that information.